usagov/usagov-2021
# USAGOV-1925: USAGOV-1925 - When the body is empty generating FAQs fails

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1925

## Description
Introduced logic that confirms the body is not empty on a Node before trying to generate FAQs.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions
Simply go to the URI of `/es/prestamos-del-gobierno` which is an example page that has the FAQ checkbox ticked on the Edit-tab, but has an empty body. The page loads, but you will see errors and stack-traces printing out in your terminal/error-log.

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
